frontend/src/app/modules/authentication/registration/registration.component.html

```diff
@@ -0,0 +1,87 @@
+<div class="registration-container">
+    <div class="registration-content">
+        <h4>Sign Up</h4>
+
+        <div class="additional-sing-up">
+            <button>Sign Up with GitHub</button>
+            <button>Sign Up with Google</button>
+        </div>
+
+        <div class="barrier">
+            <hr>
+            <p class="barrier-text">or</p>
+        </div>
+
+        <form [formGroup]="registerForm" (ngSubmit)="registerForm.valid && register()" class="registration-form"
+              action="">
+            <label>Username</label>
+            <input [formControl]="$any(registerForm.controls['username'])" type="text"
+                   placeholder="Enter your Username">
+            <div class="invalid-feedback"
+                 *ngIf="validationCheck('username', 'required')">
+                Please enter a username
+            </div>
+
+            <label>Email</label>
+            <input [formControl]="$any(registerForm.controls['email'])" type="text" placeholder="Enter your Email">
+            <div class="invalid-feedback"
+                 *ngIf="validationCheck('email', 'required')">
+                Please enter a email
+            </div>
+
+            <label>Email</label>
+            <input [formControl]="$any(registerForm.controls['firstName'])" type="text" placeholder="Enter your Email">
+            <div class="invalid-feedback"
+                 *ngIf="validationCheck('firstName', 'required')">
+                Please enter a first name
```

Comment:
I guess here should be label for 'First name', not email) Same for label for Last name

Reply:
hawk eye, I'll edit this when input component is completed